akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ClientSpec.scala

```diff
@@ -119,6 +121,47 @@ class Http2ClientSpec extends AkkaSpecWithMaterializer("""
         val (_, errorCode) = expectGOAWAY(1)
         errorCode should ===(ErrorCode.COMPRESSION_ERROR)
       }
+      "Three consecutive GET requests" in new SimpleRequestResponseRoundtripSetup {
```

Comment:
What's so special about sending three consecutive GET requests?
I don't understand the purpose of the test :-(

Reply:
Basically it tests that the examples from the HPACK spec work correctly. HPACK introduces a dependency between consecutive messages in one direction because of the compression (i.e. later requests can refer to previously sent headers to save bandwidth).

Maybe we can improve the test name to point that out more clearly.